bin/resources/GameIndex.yaml

```diff
@@ -532,6 +532,8 @@ SCAJ-20011:
   roundModes:
     eeRoundMode: 0 # Fixes Z-Fighting.
   gsHWFixes:
+    cpuSpriteRenderBW: 2 # Fixes water rendering.
+	  cpuSpriteRenderLevel: 2 # Needed for above.
```

Comment:
Can you replace the tab with 2 spaces here, and the same for every other RenderLevel line you added too.

Reply:
Done